requests_oauthlib/oauth2_session.py

```diff
@@ -289,7 +287,7 @@ def refresh_token(self, token_url, refresh_token=None, body='', auth=None,
             }
 
         r = self.post(token_url, data=dict(urldecode(body)), auth=auth,
-                      timeout=timeout, headers=headers, verify=verify)
+            timeout=timeout, headers=headers, verify=verify, withhold_token=True)
```

Comment:
Can we fix up the indenting here? You'll probably need to move withold_token to the next line to make everything fit an appropriate line length.